### PR TITLE
fix(markview-nvim): Add missing treesitter parsers

### PR DESCRIPTION
**Pull request by manuuurino**

### CODE REVIEW

File: lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua [hunk 1]
```diff
@@ -1,7 +1,15 @@
 return {
   "OXY2DEV/markview.nvim",
   ft = { "markdown", "markdown.mdx" },
-  dependencies = { "nvim-treesitter/nvim-treesitter" },
+  dependencies = {
+    "nvim-treesitter/nvim-treesitter",
+    optional = true,
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "html" })
```
Comment (Uzaaft) [inline lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua:9]:
```suggestion
        opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "html", "markdown", "markdown_inline" })
```

Comment (Uzaaft) [reply lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua:9]: These are also missing.  @manuuurino

Comment (manuuurino) [reply lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua]: oh yea, didnt think about it, thanks

**Comment by Uzaaft** [comment]
I hate formatters sometimes...

**Review by Uzaaft — APPROVED** [review]

**Comment by manuuurino** [comment]
> I hate formatters sometimes...

have you tried column width 80? imo looks good and the formater does how i would like it

**Comment by Uzaaft** [comment]
> > I hate formatters sometimes...
> 
> have you tried column width 80? imo looks good and the formater does how i would like it

brother I did that on the phone

**Pull request merged**